src/xenia/kernel/xboxkrnl/xboxkrnl_modules.cc

```diff
@@ -58,7 +59,7 @@ X_STATUS xeExGetXConfigSetting(uint16_t category, uint16_t setting,
           break;
         case 0x0009:  // XCONFIG_USER_LANGUAGE
           setting_size = 4;
-          value = 0x00000001;  // English
+          value = userlang;  // 1 - English as default
```

Comment:
We could maybe make this part of the kernel state - is there no config structure present in the kernel state as of now?